core/validatorapi/validatorapi.go

```diff
@@ -664,15 +663,28 @@ func (c Component) SubmitBeaconCommitteeSubscriptionsV2(ctx context.Context, sub
 		}
 	}
 
-	// Query BeaconCommitteeSubscriptionResponse (this is blocking).
 	var resp []*eth2exp.BeaconCommitteeSubscriptionResponse
```

Comment:
can extract this a function if you want, this parent function is already quite long.

Reply:
done